lib/tasks/demo-build.js

```diff
@@ -88,29 +89,48 @@ function buildJs(buildConfig) {
 	}
 }
 
+function loadExternalDemoData(url) {
+	return request.get({
```

Comment:
What happens in the different failure scenarios?
- URL is not valid
- Response returns a HTTP status code equal or greater than 400
- Response returns a redirect
- Response is not valid JSON

Reply:
URL is not valid: Error thrown with stack trace. Do you think it aught to be caught and the stacktrace removed as with file not found?
<img width="714" alt="screen shot 2017-09-11 at 13 48 22" src="https://user-images.githubusercontent.com/10405691/30275093-0cfdf44c-96f8-11e7-928f-2c56d0299108.png">

Response returns a HTTP status code equal or greater than 400: Error thrown with stack trace.
<img width="573" alt="screen shot 2017-09-11 at 13 51 43" src="https://user-images.githubusercontent.com/10405691/30275178-656c99b2-96f8-11e7-9075-8e83bd568f92.png">

Response returns a redirect: `request-promise-native` follows up to 10 redirects by default.

Response is not valid JSON: I've now caught this case, introducing a more helpful error.
<img width="767" alt="screen shot 2017-09-11 at 13 34 06" src="https://user-images.githubusercontent.com/10405691/30275333-e914610a-96f8-11e7-9cf8-a001ce37ebe9.png">